fdbcli/fdbcli.actor.cpp

```diff
@@ -1998,10 +1998,14 @@ ACTOR Future<bool> coordinators( Database db, std::vector<StringRef> tokens, boo
 
 ACTOR Future<bool> include( Database db, std::vector<StringRef> tokens ) {
 	std::vector<AddressExclusion> addresses;
-	if (tokens.size() == 2 && tokens[1] == LiteralStringRef("all"))
-		addresses.push_back( AddressExclusion() );
-	else {
-		for(auto t = tokens.begin()+1; t != tokens.end(); ++t) {
+	bool failed = false;
+	bool all = false;
+	for (auto t = tokens.begin() + 1; t != tokens.end(); ++t) {
+		if (*t == LiteralStringRef("all")) {
+			all = true;
+		} else if (*t == LiteralStringRef("failed")) {
```

Comment:
If you use command `include failed [address]`,  I don't think the address will be parsed and passed in? Did I miss something?

Reply:
Also if you use `include all`, why do we need to iterate through all tokens? 
In the previous code,  it is a different logic. Is there anything I don't know that was changed?

Reply:
It's in the else clause below. This entire `if` block parses token by token after the first one (which should be `include`) since it's contained in the for loop

Reply:
The extra iterating is to ensure commands like `include all failed` and `include failed all` will both work as intended. If we want to strictly limit it to one or the other, then we can insert an early `break` to avoid the iterations